reactivesocket-tck-drivers/build.gradle

```diff
@@ -26,10 +26,12 @@ dependencies {
     compile 'io.airlift:airline:0.7'
 }
 
-task (runTests, dependsOn: jar, type: JavaExec) {
+task runTests(type: JavaExec) {
     classpath(sourceSets.main.runtimeClasspath, sourceSets.main.compileClasspath)
     main = 'io.reactivesocket.tckdrivers.main.TestMain'
     args '--port',  '4567', '--serverfile',  'src/test/resources/server$.txt',  '--clientfile',  'src/test/resources/client$.txt'
 }
 
-build.finalizedBy(runTests)
+tasks.build.doLast() {
+    runTests.execute()
+}
```

Comment:
@yschimke I'm not sure the links you sent about integration tests could execute the behavior I wanted. In those cases, it seemed like the tests had to be explicitly called from the command line with `gradle <testname>`. However, I wanted it to behave like the other unit tests that run automatically during the travis build process, and this seemed like the cleanest way to do so.